apps/journeys-admin/src/components/Editor/VideoLibrary/VideoSearch/VideoSearch.tsx

```diff
@@ -2,19 +2,23 @@ import { ReactElement, useMemo, useEffect, useState, ChangeEvent } from 'react'
 import TextField from '@mui/material/TextField'
 import InputAdornment from '@mui/material/InputAdornment'
 import Search from '@mui/icons-material/Search'
+import LinkRounded from '@mui/icons-material/LinkRounded'
 import { debounce } from 'lodash'
 import Box from '@mui/material/Box'
+import { VideoBlockSource } from '../../../../../__generated__/globalTypes'
 
 interface VideoSearchProps {
   label?: string
   value?: string
   onChange: (value: string) => void
+  activeTab?: VideoBlockSource
```

Comment:
Just make this thing more generic.

```suggestion
 icon?: 'search' | 'link'
```

You'll need to update your 59 and 60 lines.